src/cwt/index.ts

```diff
@@ -0,0 +1,124 @@
+import { mdocContext } from '../../tests/context'
+import { cborDecode, cborEncode } from '../cbor'
+import {
+  type CoseKey,
+  Mac0,
+  type Mac0Options,
+  type Mac0Structure,
+  Sign1,
+  type Sign1Options,
+  type Sign1Structure,
+} from '../cose'
+import { CoseType } from '../cose'
+
+type Header = {
+  protected?: Record<string, unknown>
+  unprotected?: Record<string, unknown>
+}
+
+type CWTOptions = {
+  type: CoseType
+  key: CoseKey
+}
+
+interface CWTVerifyOptions {
+  type: CoseType
+  token: Uint8Array
+  key?: CoseKey
+}
+
+enum CwtStandardClaims {
+  Iss = 1,
```

Comment:
I think we should be concistent in how we name enum keys. Here you use the three letter abbreviated keys., but in the satus list you use the full word `Type` (which abbreviated is `Typ`). Consistency is important.

Also we should probably move the CwtProtectedHeaders enum here as well?